### DISCUSSION TITLE
USB-Direct on V2

### DISCUSSION THREAD

**Discussion by clee604** [gtxaspec/wz_mini_hacks#654] [Q&A] [answered]
Hi,

Has anyone tried using usb-direct (or the webcam mode) with a V2 camera? I have it currently working with a USB ethernet adapter.

The main reason I ask is because unlike the V3, the USB ethernet is connected to the USB A port on the back of the V2, and another microusb cable provides the power (the V3 uses a OTG cable instead).

I'm thinking about switching to USB-Direct to save on a cable/usb adapter, but I'm not sure if it works the same as a V3 (single cable via Microusb) or the V2 differs due to the USB A port on the back.

**Comment by gtxaspec** [comment, marked as answer]
On the v2, micro USB is power only. No data

**Reply by clee604** [reply]
Thanks @gtxaspec - in that case to get USB-Direct or Webcam mode to work I need to use a USB A to USB A cable?

**Reply by gtxaspec** [reply]
Yes, that's correct

**Reply by clee604** [reply]
Thanks!
